src/scripts/towns/Town.ts

```diff
@@ -1210,6 +1210,9 @@ const AccumulaTownShop = new Shop([
     ItemList['Dusk_stone'],
     ItemList['Shiny_stone'],
 ]);
+const AnvilleTownShop = new Shop([
+    ItemList['Meloetta_(pirouette)'],
```

Comment:
Just for consistency with the other Pokémon shop items:
```suggestion
    ItemList['Meloetta (pirouette)'],
```